be-python-flask: build fails when "python-ldap" is added to requirements.txt

Quick starter: be-python-flask
Build fails when "python-ldap" is added to requirements.txt
Error message:  unable to execute 'gcc': No such file or directory

  building '_ldap' extension
  creating build/temp.linux-x86_64-3.7
  creating build/temp.linux-x86_64-3.7/Modules
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DHAVE_SASL -DHAVE_TLS -DHAVE_LIBLDAP_R -DHAVE_LIBLDAP_R -DLDAPMODULE_VERSION=3.2.0 -DLDAPMODULE_AUTHOR=python-ldap project -DLDAPMODULE_LICENSE=Python style -IModules -I/usr/local/include/python3.7m -c Modules/LDAPObject.c -o build/temp.linux-x86_64-3.7/Modules/LDAPObject.o
  unable to execute 'gcc': No such file or directory
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for python-ldap
  Running setup.py clean for python-ldap
Successfully built flask-jwt-extended
Failed to build python-ldap

However, gcc should be available? (is part of build-essential)

Tried the above suggestion and made one step further, but build still fails:

building '_ldap' extension
    creating build/temp.linux-x86_64-3.7
    creating build/temp.linux-x86_64-3.7/Modules
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DHAVE_SASL -DHAVE_TLS -DHAVE_LIBLDAP_R -DHAVE_LIBLDAP_R -DLDAPMODULE_VERSION=3.2.0 -DLDAPMODULE_AUTHOR=python-ldap project -DLDAPMODULE_LICENSE=Python style -IModules -I/usr/local/include/python3.7m -c Modules/LDAPObject.c -o build/temp.linux-x86_64-3.7/Modules/LDAPObject.o
    In file included from Modules/LDAPObject.c:8:0:
    Modules/constants.h:7:18: fatal error: lber.h: No such file or directory
     #include "lber.h"
                      ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1

I already added some additional packages which I could find via Google,

        libssl-dev \
        libsasl2-dev \
        libldap2-dev \

but it still fails with above error. Any ideas what package is missing for successful build?

This did not help either. Same error:
Modules/constants.h:7:18: fatal error: lber.h: No such file or directory
#include "lber.h"
^
compilation terminated.
error: command 'gcc' failed with exit status 1

--- a/boilerplates/be-python-flask/files/docker/Dockerfile
+++ b/boilerplates/be-python-flask/files/docker/Dockerfile
@@ -1,6 +1,6 @@
 FROM python:3.7.0-slim
 
-MAINTAINER "Andras Gyacsok" <andras.gyacsok@boehringer-ingelheim.com>
+LABEL maintainer="Andras Gyacsok <andras.gyacsok@boehringer-ingelheim.com>"
 
 # Never prompts the user for choices on installation/configuration of packages
 ENV DEBIAN_FRONTEND noninteractive
@@ -10,7 +10,6 @@ ARG projectHome=/opt/app-root/src
 ARG home=${projectHome}
 ARG nexusHostWithBasicAuth
 ARG nexusHostWithoutScheme
-
 
 # Define en_US.
 ENV LANGUAGE en_US.UTF-8
@@ -32,7 +31,7 @@ RUN set -ex  && \
         m4 \
         automake \
         gettext \
-    ' \
+    ' && \
     apt-get update -yqq && \
     apt-get upgrade -yqq && \
     apt-get install -yqq --no-install-recommends \
@@ -49,7 +48,6 @@ RUN set -ex  && \
     # CLEAN UP
     apt-get autoremove -yqq --purge && \
     apt-get clean
-
 
 COPY run.sh ${projectHome}
 COPY dist ${projectHome}

